pkgs/tools/games/gog-downloader/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, makeWrapper, lib, php }:
+
+let
+  pname = "gog-downloader";
+  version = "1.2.3";
+  hash = "0kv09vmrz6dcjvaf2sw89cpb14wdr01r1kkzg84szfxwzr5bcc9p";
+in
+stdenv.mkDerivation {
+  inherit pname version hash;
+
+  src = fetchurl {
+    url = "https://github.com/RikudouSage/GogDownloader/releases/download/v${version}/gog-downloader";
+    sha256 = hash;
```

Comment:
Why? It works the same, but in my version all parameters that will change often are in one place.

Reply:
I didn't say that "your version" was bad or not working. But I'm all in favor of consistency so, let's be consistent on how it's done on nixpkgs and do as I advised, for each comment in here.

Reply:
Yeah, consistency, sure. Side note, "do as I say because I said so" doesn't really help people understand your point.

Reply:
Feel free to let me know better wording here, I'm all ears.

Reply:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.